### PR TITLE
Add logging statements for debugging cuttlefish cloud

### PR DESCRIPTION
**Pull request by marktefftech**

### CODE REVIEW

File: src/clusterfuzz/_internal/platforms/android/flash.py [hunk 1]
```diff
@@ -122,16 +124,21 @@ def flash_to_latest_build_if_needed():
...
         target = build_params.get('target') + FLASH_DEFAULT_BUILD_TARGET
+        logs.log_warn('is_android_cuttlefish() returned True. Target: %s.' % target)
       else:
         target = build_params.get('target') + '-userdebug'
+        logs.log_warn('is_android_cuttlefish() returned False. Target: %s.' % target)
 
       # Cache target in environment. This is also useful for cases when
       # device is bricked and we don't have this information available.
       environment.set_value('BUILD_TARGET', target)
+    logs.log_warn('build_params not found.')
```
Comment (TheCoryBarker) [inline src/clusterfuzz/_internal/platforms/android/flash.py:141]: will this always be hit? Even if build_params are found?

File: src/clusterfuzz/_internal/platforms/android/flash.py [hunk 2]
```diff
@@ -64,7 +64,9 @@ def download_latest_build(build_info, image_regexes, image_directory):
   # download it.
   build_id = build_info['bid']
   target = build_info['target']
+  logs.log_warn('target stored in current build_info: %s.' % target)
```
Comment (TheCoryBarker) [inline src/clusterfuzz/_internal/platforms/android/flash.py:67]: I think we just want logs.log for all these logs instead of logs.log_warn

Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/platforms/android/flash.py:67]: +1

**Review by jonathanmetzman — COMMENTED** [review]
Change to log instead of log_warn

File: src/clusterfuzz/_internal/platforms/android/flash.py (hunk 1 above)
Comment (marktefftech) [reply src/clusterfuzz/_internal/platforms/android/flash.py]: Fixed.

File: src/clusterfuzz/_internal/platforms/android/flash.py (hunk 2 above)
Comment (marktefftech) [reply src/clusterfuzz/_internal/platforms/android/flash.py]: Done

**Comment by marktefftech** [comment]
Done

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Comment by jonathanmetzman** [comment]
Gonna wait for the tests to pass.

**Pull request merged**